be/src/exec/spill/mem_table.h

```diff
@@ -59,8 +59,8 @@ class SpillableMemTable {
     virtual bool is_empty() = 0;
     size_t mem_usage() { return _tracker->consumption(); }
     // append data to mem table
-    virtual Status append(ChunkPtr chunk) = 0;
-    virtual Status append_selective(const Chunk& src, const uint32_t* indexes, uint32_t from, uint32_t size) = 0;
+    virtual void append(ChunkPtr chunk) = 0;
```

Comment:
append-related code needs support return Status code

Reply:
> append-related code needs support return Status code

why? actually, it always return OK for now.

Reply:
We may later use partial ordering in the append function and may need to handle the Status return result